docs/readme.md

```diff
@@ -1,21 +1,27 @@
 # AGAGD
 
 ## Overview
-An app to display users, games, tournaments, and ratings information on the AGA website.  
+An app to display users, games, tournaments, and ratings information on the AGA website.
 A port of the old eurogo Games Database to python, for the AGA
 
  - agagd_core/     --
- - agagd/          -- 
+ - agagd/          --
 
 ## Development
 
+### Code Style
+The AGAGD project uses pre-commit for some basic linting, so before it is required to have pre-commit installed and any errors fixed before making a pull request. ([Pre Commit](https://pre-commit.com/), [DJango Code Style](https://docs.djangoproject.com/en/3.2/internals/contributing/writing-code/coding-style/))
+
+#### Install pre-commit
+* `shell> pip3 install pre-commit`
+* `shell> pre-commit install`
```

Comment:
Not strictly needed for developers since it gets checked by GitHub, right?  Also, the documentation is on `pre-commit` site.

Reply:
May be, but I think for now it is fine.